configs/config.dell_optiplex_9010_sff_txt

```diff
@@ -1,13 +1,27 @@
+CONFIG_LOCALVERSION="v0.1.0-rc3"
+CONFIG_OPTION_BACKEND_NONE=y
 CONFIG_VENDOR_DELL=y
-CONFIG_CBFS_SIZE=0x400000
 CONFIG_ONBOARD_VGA_IS_PRIMARY=y
 CONFIG_SEABIOS_PS2_TIMEOUT=3000
+CONFIG_PXE_ROM_ID="8086,1502"
 CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x100000
 CONFIG_PCIEXP_CLK_PM=y
 CONFIG_TPM_MEASURED_BOOT=y
-CONFIG_HIDE_MEI_ON_ERROR=y
+# CONFIG_RESOURCE_ALLOCATION_TOP_DOWN is not set
+# CONFIG_USE_PC_CMOS_ALTCENTURY is not set
 CONFIG_INTEL_TXT=y
 CONFIG_INTEL_TXT_BDR_VERSION=4
-CONFIG_POST_DEVICE_LPC=y
+CONFIG_INTEL_TXT_LOGGING=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_8=y
 CONFIG_SEABIOS_USB_SIGATT_TIME=3000
 CONFIG_SEABIOS_ADD_SERCON_PORT_FILE=y
+CONFIG_PXE=y
+CONFIG_BUILD_IPXE=y
+# CONFIG_IPXE_SERIAL_CONSOLE is not set
+CONFIG_IPXE_ADD_SCRIPT=y
+CONFIG_IPXE_SCRIPT="3rdparty/dasharo-blobs/dasharo/dasharo.ipxe"
+CONFIG_INTEL_TXT_BIOSACM_FILE="IVB_BIOSAC_PRODUCTION.BIN"
+CONFIG_INTEL_TXT_SINITACM_FILE="SNB_IVB_SINIT_20190708_PW.bin"
+CONFIG_SMSC_SCH5545_EC_FW_FILE="sch5545_ecfw.bin"
```

Comment:
These blobs are being added in DTS during deployment/update. They should not be here in the config.

Reply:
yes, I'll only leave them in for the WIP debug phase for convenience